### PR TITLE
Docs: Add refrences to granite model to community docs

### PR DESCRIPTION
**Pull request by kelbrown20**
**Description:** Adding info and references to the granite model to the community repo

### CODE REVIEW

**Review by lhawthorn — CHANGES_REQUESTED** [review]
Looks good to me so far. Would appreciate review of my comments and either updates or helping with my understanding. Will not merge until you give me the thumbs up, too, as the original PR indicates not complete.

File: FAQ.md [hunk 1]
```diff
@@ -100,19 +101,25 @@ A list of common problems associated with downloading the `ilab` CLI tool can be
 
 ### Why should I contribute?
 
-InstructLab is designed to enable collaboration around Merlinite-7b, an open source LLM that contributors can access through [Hugging Face](https://huggingface.co/collections/ibm/large-language-models-65d8bb0fa7d935c9c49b4104). Participating is an opportunity to contribute to open source AI regardless of technical background.
+InstructLab is designed to enable collaboration around Merlinite-7b and Granite-7b, an open source licensed LLM that contributors can access through [Hugging Face](https://huggingface.co/collections/ibm/large-language-models-65d8bb0fa7d935c9c49b4104). Participating is an opportunity to contribute to open source AI regardless of technical background.
 
 When contributors write an extension to the existing taxonomy, make a pull request, and get it reviewed and merged, their changes are rolled out in the next build. This update strategy expedites the model’s capabilities and allows contributors to see the impact that they have made on the model much sooner than other LLMs.
```
Comment (JamesKunstle) [inline FAQ.md:106]: r/extension/addition

File: FAQ.md [hunk 2]
```diff
@@ -100,19 +101,25 @@ A list of common problems associated with downloading the `ilab` CLI tool can be
...
 
 ### What large language models (LLMs) am I contributing to through the InstructLab project?
 
-Contributions to the InstructLab project update Merlinite-7b, an open source LLM. Contributors have direct access to the model they are improving through [Hugging Face](https://huggingface.co/collections/ibm/large-language-models-65d8bb0fa7d935c9c49b4104).
+Contributions to the InstructLab project update Merlinite-7b or Granite-7b, an open source licensed LLM. Contributors have direct access to the model they are improving through [Hugging Face](https://huggingface.co/collections/ibm/large-language-models-65d8bb0fa7d935c9c49b4104).
```
Comment (JamesKunstle) [inline FAQ.md:110]: r/update/fine-tune

File: FAQ.md [hunk 3]
```diff
@@ -100,19 +101,25 @@ A list of common problems associated with downloading the `ilab` CLI tool can be
...
 
-Merlinite-7b is a Mistral-7b derivative model trained with the LAB (**L**arge-scale **A**lignment for chat**B**ots) using Mixtral-8x7b-Instruct as a teacher model.
+Merlinite-7b is a Mistral-7b derivative model fine tuned with the LAB (**L**arge-scale **A**lignment for chat**B**ots) process using Mixtral-8x7b-Instruct as a teacher model.
 
-More information about the Merlinite-7b can be found on the [Hugging Face project page](https://huggingface.co/collections/ibm/large-language-models-65d8bb0fa7d935c9c49b4104).
+More information about the Merlinite-7b can be found on the [Hugging Face project page](https://huggingface.co/instructlab/merlinite-7b-lab).
+
+### What is Granite-7?
+
+Granite-7b is a model that was build from scratch and fine tuned with the LAB (**L**arge-scale **A**lignment for chat**B**ots) process  .
```
Comment (JamesKunstle) [inline FAQ.md:120]: r/build/built, `from scratch by IBM`

File: FAQ.md [hunk 4]
```diff
@@ -188,7 +195,7 @@ After a pull request is accepted, the changes are regularly incorporated into In
 
 ### What is the software license for InstructLab?
 
-InstructLab and the Merlinite-7b project are distributed under [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0).
+The InstructLab project and the Merlinite-7b & Granite-7b models are distributed under [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0).
```
Comment (JamesKunstle) [inline FAQ.md:198]: "The InstructLab project and the Merlinite-7b and Granite-7b models"

**Review by JamesKunstle — CHANGES_REQUESTED** [review]
Read through the changes, made a few requests for changes in form / specificity.

File: FAQ.md [hunk 5]
```diff
@@ -100,19 +101,25 @@ A list of common problems associated with downloading the `ilab` CLI tool can be
 
 ### Why should I contribute?
 
-InstructLab is designed to enable collaboration around Merlinite-7b, an open source LLM that contributors can access through [Hugging Face](https://huggingface.co/collections/ibm/large-language-models-65d8bb0fa7d935c9c49b4104). Participating is an opportunity to contribute to open source AI regardless of technical background.
+InstructLab is designed to enable collaboration around Merlinite-7b and Granite-7b, an open source licensed LLM that contributors can access through [Hugging Face](https://huggingface.co/collections/ibm/large-language-models-65d8bb0fa7d935c9c49b4104). Participating is an opportunity to contribute to open source AI regardless of technical background.
```
[the remark this answers is not in the document]
Comment (kelbrown20) [reply FAQ.md:104]: Yes I agree with that, I'll update this link

**Comment by kelbrown20** [comment]
@lhawthorn Can you PTAL when you get the chance?

**Review by lhawthorn — APPROVED** [review]
LGTM

**Comment by bjhargrave** [comment]
@kelbrown20 Can you rebase on main? There is a new action to spell check.

File: FAQ.md [hunk 6]
```diff
@@ -15,6 +15,7 @@ Last updated: April 2024
   - [Why should I contribute?](#why-should-i-contribute)
   - [What large language models (LLMs) am I contributing to through the InstructLab project?](#what-large-language-models-llms-am-i-contributing-to-through-the-instructlab-project)
   - [What is Merlinite-7b?](#what-is-merlinite-7b)
+  - [What is Granite-7b-lab](#what-is-granite-7-lab)
```
Comment (nathan-weinberg) [inline FAQ.md:18]:
```suggestion
  - [What is Granite-7b-lab?](#what-is-granite-7-lab)
```

File: FAQ.md [hunk 7]
```diff
@@ -70,7 +71,7 @@ LAB (**L**arge-scale **A**lignment for chat**B**ots) is a novel synthetic data-b
 
 The LAB approach allows incrementally adding new knowledge and skills to an already pre-trained model without catastrophic forgetting.
 
-More information about the LAB method can be found on this [Hugging Face project page](https://huggingface.co/ibm/merlinite-7b).
+More information about the LAB method can be found on this [Hugging Face project page](https://huggingface.co/instructlab).
```
Comment (nathan-weinberg) [inline FAQ.md:74]:
```suggestion
More information about the LAB method can be found on the [Hugging Face project page](https://huggingface.co/instructlab).
```

File: FAQ.md [hunk 8]
```diff
@@ -100,19 +101,25 @@ A list of common problems associated with downloading the `ilab` CLI tool can be
...
 
 ### What large language models (LLMs) am I contributing to through the InstructLab project?
 
-Contributions to the InstructLab project update Merlinite-7b, an open source LLM. Contributors have direct access to the model they are improving through [Hugging Face](https://huggingface.co/collections/ibm/large-language-models-65d8bb0fa7d935c9c49b4104).
+Contributions to the InstructLab project fine-tune Merlinite-7b or Granite-7b, an open source licensed LLM. Contributors have direct access to the model they are improving through [Hugging Face](https://huggingface.co/instructlab).
 
 ### What is Merlinite-7b?
 
-Merlinite-7b is a Mistral-7b derivative model trained with the LAB (**L**arge-scale **A**lignment for chat**B**ots) using Mixtral-8x7b-Instruct as a teacher model.
+Merlinite-7b is a Mistral-7b derivative model fine tuned with the LAB (**L**arge-scale **A**lignment for chat**B**ots) process using Mixtral-8x7b-Instruct as a teacher model.
```
Comment (nathan-weinberg) [inline FAQ.md:114]:
```suggestion
Merlinite-7b is a Mistral-7b derivative model fine tuned with the LAB (**L**arge-scale **A**lignment for chat**B**ots) method using Mixtral-8x7b-Instruct as a teacher model.
```

File: FAQ.md [hunk 9]
```diff
@@ -100,19 +101,25 @@ A list of common problems associated with downloading the `ilab` CLI tool can be
...
 
-More information about the Merlinite-7b can be found on the [Hugging Face project page](https://huggingface.co/collections/ibm/large-language-models-65d8bb0fa7d935c9c49b4104).
+More information about the Merlinite-7b can be found on the [Hugging Face project page](https://huggingface.co/instructlab/merlinite-7b-lab).
+
+### What is Granite-7-lab?
+
+Granite-7b-lab is a model that was built from scratch by IBM and fine tuned with the LAB (**L**arge-scale **A**lignment for chat**B**ots) process.
```
Comment (nathan-weinberg) [inline FAQ.md:120]:
```suggestion
Granite-7b-lab is a model that was built from scratch by IBM and fine tuned with the LAB (**L**arge-scale **A**lignment for chat**B**ots) method.
```

File: FAQ.md [hunk 10]
```diff
@@ -187,7 +194,7 @@ After a pull request is accepted, the changes are regularly incorporated into In
 
 ### What is the software license for InstructLab?
 
-InstructLab and the Merlinite-7b project are distributed under [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0).
+The InstructLab project and the Merlinite-7b and Granite-7b models are distributed under [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0).
```
Comment (nathan-weinberg) [inline FAQ.md:197]:
```suggestion
The InstructLab project as well as the Merlinite-7b and Granite-7b models are distributed under [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0).
```

File: README.md [hunk 11]
```diff
@@ -57,7 +57,7 @@ There are teams of contributors from Red Hat and IBM that will review your pull
 
 ### See your contributions impact an AI model
 
-The IBM model `Merlinite` builds regularly. Sometime after your pull request is merged, Merlinite is updated and you can see locally that the model improved with the skill or knowledge you taught it.
+The models Merlinite-7b and Granite-7b builds regularly. Sometime after your pull request is merged, Merlinite is updated and you can see locally that the model improved with the skill or knowledge you taught it.
```
Comment (nathan-weinberg) [inline README.md:60]:
```suggestion
The Merlinite-7b and Granite-7b models are built regularly. Sometime after your pull request is merged, Merlinite is updated and you can see locally that the model improved with the skill or knowledge you taught it.
```

Comment (kelbrown20) [reply README.md:60]: I usually prefer active voice when I can, but lemme see if I can re word something

Comment (nathan-weinberg) [reply README.md]: Honestly just an idea from me - I trust your judgement!

**Pull request merged**